tests/x11/chrome.pm

```diff
@@ -0,0 +1,36 @@
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    my $CHROME_URL = "https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm";
+
+    mouse_hide;
+
+    x11_start_program("xterm");
+    assert_screen('xterm-started');
+
+    script_sudo "zypper -n install $CHROME_URL";
+    save_screenshot;
```

Comment:
no handling of exit status? script_sudo only checks for idle machine - which is a bit weak